src/pages/AdminPage.jsx

```diff
@@ -293,8 +121,32 @@ const AdminPage = () => {
   const [isDateAscend, setDateAscend] = useState(false);
   const [isVolunteerAscend, setVolunteerAscend] = useState(false);
   const [isApprovalAscend, setApprovalAscend] = useState(false);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [data, setData] = useState(dummy);
+  const [change, setChange] = useState(true);
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'setChange' is assigned a value but never used.